flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
so no on aarch64?

Reply:
Currently we do not build nor test on aarch64, however there are some plans for the future to support it.

You can see all the artifacts we build here: https://github.com/ruffle-rs/ruffle/releases

Reply:
If it is built from source, you can remove this file?

Reply:
Sure, removed! Although I haven't tested it yet to see if it builds and works properly on aarch64

Reply:
I tried building it on Aarch64 Linux and it succeeded